### DISCUSSION TITLE
tell me how to implement the Assembler

### DISCUSSION THREAD

**Discussion by ghost** [community/community#109726] [Programming Help]
### Body

Assembler implementation.****

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by davea42** [comment]
Without more detailed information from you ... it seems impossible to help.
An assembler written in...python? 
An assembler for a new CPU type?  Something else entirely?

**Reply by ghost** [reply]
The X86 architecture.

**Comment by gbpii** [comment]
**Useful links for you:**

The Holy Book of X86, A simple guide to x86 architecture, assembly, memory management, paging, segmentation, interrupts, etc.
https://github.com/Captainarash/The_Holy_Book_of_X86

Intro to x86 Assembly Language (Part 1):
https://www.youtube.com/watch?v=wLXIWKUWpSs

x86 Assembly/X86 Architecture:
https://en.wikibooks.org/wiki/X86_Assembly/X86_Architecture
